docs/src/user/gotchas.md

```diff
@@ -1,13 +1,69 @@
 # Gotchas
 
-Here are some potential "gotchas" for interpreter authors:
+Yk has some unusual implications, requirements and limitations that interpreter
+authors should be aware of.
+
+## Trace quality depends upon completeness of interpreter IR.
+
+Yk works by recording control flow paths (traces) through your interpreter
+implementation. Each trace ends up as an ordered list of LLVM IR blocks which
+are "stitched together" and compiled. During the trace collection process, by
+default calls are inlined (if possible).
+
+Generally speaking (but not always), the more the JIT can inline, the more
+optimal the JITted code will be. In order to inline calls, the JIT needs to
+have LLVM IR for callees. Upon encountering a call to a target for which there
+is no IR, the JIT will be forced to outline (i.e. call) the target.
+
+Yk uses fat LTO to collect (and embed into the resulting binary) a
+"full-program" IR for your interpreter. `yk-config` provides the relevant clang
+flags to make this happen. You should make sure that the build system of your
```

Comment:
s/of your/for your/

Reply:
Both work for me. Don't feel all that strongly.

Reply:
Nor I. Let's drop this one.